<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAnaTypeDialog</class>
 <widget class="QDialog" name="CAnaTypeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="m_radioButton0">
         <property name="text">
          <string>未定义</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton6">
         <property name="text">
          <string>视在功率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton12">
         <property name="text">
          <string>温度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton18">
         <property name="text">
          <string>故障电压</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="m_radioButton1">
         <property name="text">
          <string>电流</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton7">
         <property name="text">
          <string>功率因素</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton13">
         <property name="text">
          <string>直流</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton19">
         <property name="text">
          <string>故障电流</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="m_radioButton2">
         <property name="text">
          <string>电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton8">
         <property name="text">
          <string>直流电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton14">
         <property name="text">
          <string>脉冲</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton20">
         <property name="text">
          <string>零序电流</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QRadioButton" name="m_radioButton3">
         <property name="text">
          <string>站用变电压</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton9">
         <property name="text">
          <string>直流电流</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton15">
         <property name="text">
          <string>幅值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton21">
         <property name="text">
          <string>差动电流</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QRadioButton" name="m_radioButton4">
         <property name="text">
          <string>有功功率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton10">
         <property name="text">
          <string>档位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton16">
         <property name="text">
          <string>设备态</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton22">
         <property name="text">
          <string>故障测距</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QRadioButton" name="m_radioButton5">
         <property name="text">
          <string>无功功率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton11">
         <property name="text">
          <string>频率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton17">
         <property name="text">
          <string>阻抗</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_radioButton23">
         <property name="text">
          <string>零序差压</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CAnaTypeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CAnaTypeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
